tiled/client/base.py

```diff
@@ -172,6 +172,11 @@ def uri(self):
         "Direct link to this entry"
         return self.item["links"]["self"]
 
+    @property
+    def structure_family(self):
+        "Quick access to this entry"
+        return self.item["attributes"]["structure_family"]
```

Comment:
Sorry, one more note that I didn't think of in the first pass. I think it would be nice if this returned the `StructureFamily` enum, e.g. `StructureFamily.array` instead of the string `'array'`.